functions/src/endpoint_paths/point_log.ts

```diff
@@ -34,25 +34,25 @@ logs_app.use(firestoreTools.validateFirebaseIdToken)
 
 /**
  * Handle a PointLog
+ * 
+ *  @throws  422 - MissingRequiredParameters
+ *  @throws  426 - IncorrectFormat
```

Comment:
Can you update the comment to include all parameters in the format `@param body.log_id - id of the Point Log to handle.`

Can you also go through each of the functions that the endpoint calls and add the errors that the functions throw to the comments? At the very least, each endpoint could throw 400 - Unknown User, 401 - Unauthorized, and 500 -Server Error